test/e2e/config/e2e_conf.yaml

```diff
@@ -130,8 +130,7 @@ variables:
 
   INIT_WITH_BINARY: "https://github.com/kubernetes-sigs/cluster-api/releases/download/${CAPI_FROM_RELEASE}/clusterctl-{OS}-{ARCH}"
   # Pin Calico version
-  CALICO_PATCH_RELEASE: "v3.21.0"
-  CALICO_MINOR_RELEASE: "v3.21"
+  CALICO_PATCH_RELEASE: "v3.24.1"
   # Pin CertManager for upgrade tests
   CERT_MANAGER_RELEASE: v1.5.3
```

Comment:
I was referring to this in https://github.com/metal3-io/cluster-api-provider-metal3/pull/598#discussion_r995542862
and since it is merged now, what should we do with this? can we bump it to v1.9.1? 

/cc @mboukhalfa

Reply:
I have not tested a scenario where we install **newer** cert manager manually then `clusterctl init` tries to install older one. if it works than we can uplift else this is the version that `clusterctl` will install with `capi v1alpha4` at the end, we do not need to uplift it.